CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.5.0...master
 
+### Added
```

Comment:
If we want to *add* this as a new contract, we should attach it to the public javadoc, e.g. in `ViewIssueAction`, describing the promised behavior.

Reply:
AFAIK we don't want it as a _contract_. Let's delete this entry @drspartez

Reply:
If we don't consider it a behavioral contract then we can remove it at any point. It also means that the users can use these new observations *if* they're present, but they might go missing at any point.
We mark these as `### Fixed`.